lib/client/layout.js

```diff
@@ -9,6 +9,10 @@ BlazeLayout.setRoot = function(root) {
 };
 
 BlazeLayout.render = function render(layout, regions) {
+  // 'layout' should be null (to render nothing) or an existing template name
+  if (layout !== null && !Template[layout])
```

Comment:
Please use the new `BlazeLayout._getTemplate` instead of accessing `Template[layout]` directly. See https://github.com/kadirahq/blaze-layout/pull/50 for more information. (Probably you can just move call to the `BlazeLayout._getTemplate` to the top.)